apps/src/Observer.js

```diff
@@ -0,0 +1,52 @@
+/** @file Helper for consuming ObservableEvents. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
```

Comment:
[funcscope: true](http://jshint.com/docs/options/#funcscope): Passes, enabling globally.

[newcap: true](http://jshint.com/docs/options/#newcap): Seems this option is deprecated, removing it from my 'strict list.'  Only five offenses though.

[nonew: true](http://jshint.com/docs/options/#nonew): Currently there are five offenses, will consider fixing.

[shadow: false](http://jshint.com/docs/options/#shadow): Passes, enabling globally.

[unused: true](http://jshint.com/docs/options/#unused): 559 errors.  Not touching this one for now, but I find it really useful in keeping my own code clean.

[eqeqeq: true](http://jshint.com/docs/options/#eqeqeq): 207 errors.  Also leaving as a local rule for now.

 [maxlen](http://jshint.com/docs/options/#maxlen): 90 characters fails all over the place.  Even 120, which is the max allowed by [our styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages), fails more than 400 times, often due to long XML strings in tests.  This stays local for now.

[maxparams](http://jshint.com/docs/options/#maxparams): 30 errors at 5 params, 8 errors at 6 params, 1 error at 7.  8 params passes on our current code.  Can we reasonably say we should complain if a function has more than 8 parameters?  I'd still like to keep a stricter local rule.

[maxstatements](http://jshint.com/docs/options/#maxstatements): 200 currently passes.  We have six functions in apps that are more than 100 statements long.